### PR TITLE
Add WCSAxes dependency for doc builds

### PR DESCRIPTION
**Pull request by patti**
An example was added to `docs/wcs/index.rst`, shown here:
wcsaxes_example

### CODE REVIEW

File: docs/wcs/index.rst [hunk 1]
```diff
@@ -235,6 +235,29 @@ ability to use the :class:`~astropy.wcs.WCS` to define projections in
 Matplotlib. More information on installing and using WCSAxes can be found `here
 <http://wcsaxes.readthedocs.org>`__.
 
+.. plot::
+    :include-source:
+
+    import numpy as np
```
Comment (adonath) [inline docs/wcs/index.rst]: I think this in an unused import

File: .continuous-integration/travis/setup_dependencies_common.sh [hunk 2]
```diff
@@ -47,6 +47,7 @@ fi
 if [[ $SETUP_CMD == build_sphinx* ]]
 then
   $CONDA_INSTALL Sphinx=1.2.2 Pygments matplotlib
+  pip install wcsaxes
```
Comment (adonath) [inline .continuous-integration/travis/setup_dependencies_common.sh]: As I don't have much experience with this, @astrofrog Is this ok?

File: docs/wcs/index.rst [hunk 3]
```diff
@@ -235,6 +235,29 @@ ability to use the :class:`~astropy.wcs.WCS` to define projections in
...
+    from matplotlib import rcParams, pyplot as plt
+    from astropy.io import fits
+    from astropy.wcs import WCS
+    from astropy.visualization import astropy_mpl_style
+    rcParams.update(astropy_mpl_style)
```
Comment (adonath) [inline docs/wcs/index.rst]: Once  #4031 is merged, you don't have to set `astropy_mpl_style` explicitly anymore in the docs.

Comment (patti) [inline docs/wcs/index.rst]: Nice, I was wondering about that.

File: docs/wcs/index.rst (hunk 1 above)
Comment (patti) [inline docs/wcs/index.rst]: You're right. I removed it at one point but it got caught in a copy/paste. I'll remove it.

**Comment by embray** [comment]
I think this should just have a mention in the changelog under "Other Changes"--just that WCSAxes is now required to build the Astropy docs.

**Comment by astrofrog** [comment]
:+1: 

We should switch to one of the new Matplotlib colormaps once those are available :)

File: docs/wcs/index.rst (hunk 3 above)
Comment (patti) [inline docs/wcs/index.rst]: @adonath - Should I go ahead and remove that then?

File: .continuous-integration/travis/setup_dependencies_common.sh (hunk 2 above)
Comment (astrofrog) [inline .continuous-integration/travis/setup_dependencies_common.sh]: Yep!

**Comment by astrofrog** [comment]
Thanks @patti!

**Comment by astrofrog** [comment]
Oops I forgot we need to add wcsaxes to the ``rtd-pip-requirements`` file. Done in 23d937cc0dd49183e6964125148a0beeb6422262.

**Pull request merged**